pyproject.toml

```
@@ -43,7 +43,6 @@ dependencies = [
     "matplotlib >= 3.4",
     "numpy >= 1.18",
     "pandas >= 1.2",
-    "scikit_learn >= 0.23",
     "scipy >= 1.5",
     "seaborn >= 0.11.0"
```

Need sqlalchemy and psycopg2 in project dependencies
```suggestion
    "psycopg2 >= 2.9",
    "seaborn >= 0.11.0",
    "sqlalchemy >= 1.4",
```